README.md

```diff
@@ -3,6 +3,8 @@ API Server for CogniCity
 
 [![Build Status](https://travis-ci.org/urbanriskmap/cognicity-server.svg?branch=master)](https://travis-ci.org/urbanriskmap/cognicity-server)
 
+[![Coverage Status](https://coveralls.io/repos/github/urbanriskmap/cognicity-server/badge.svg?branch=server-object-refactor)](https://coveralls.io/github/urbanriskmap/cognicity-server?branch=server-object-refactor)
+
```

Comment:
just saw you are doing that in a later test for some of the endpoints. Maybe we should have a conversation about how we can roll this out a bit more widely (rather than just having query statements in the beginning of the code).